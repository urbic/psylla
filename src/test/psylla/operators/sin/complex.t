0 1 complex sin ?
