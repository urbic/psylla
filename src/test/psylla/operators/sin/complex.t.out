0.0+1.1752011936438014i
